webdriver/tests/bidi/browsing_context/close/prompt_unload.py

```diff
@@ -0,0 +1,87 @@
+import pytest
+import asyncio
+
+from webdriver.bidi.modules.input import Actions
+
+pytestmark = pytest.mark.asyncio
+
+CONTEXT_DESTROYED_EVENT = "browsingContext.contextDestroyed"
+USER_PROMPT_OPENED_EVENT = "browsingContext.userPromptOpened"
+
+
+@pytest.mark.parametrize("prompt_unload", [None, False])
+async def test_prompt_unload_not_triggering_dialog(bidi_session, inline, subscribe_events, top_context, prompt_unload):
+    url = inline(
+        "<script>window.addEventListener('beforeunload', event => {event.preventDefault();});</script>")
+
```

Comment:
To help with beforeunload test I'm going to add a new HTML support file via https://phabricator.services.mozilla.com/D198682#change-kOn5Z3ucC3Tn. We could use that one instead of specifying the HTML content for each and every test? I could move this patch to this repository so that we can get it landed already today if that helps.

Reply:
FYI this is now awaiting review on https://github.com/web-platform-tests/wpt/pull/44232/

Reply:
I will wait for that to simplify this one.

Reply:
The PR has been merged now so you can go ahead.

Reply:
Hi @Lightning00Blade. I wonder if you have an update regarding these tests. Thanks.

Reply:
I was OOO addressed the comments you had.